src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -1372,8 +1374,13 @@ public void TakeScreenshot(string path)
 			AddOnScreenMessage($"{fi.Name} saved.");
 		}
 
-		public void FrameBufferResized()
+		/// <param name="forceWindowResize">Override <see cref="Common.Config.ResizeWithFramebuffer"/></param>
+		public void FrameBufferResized(bool forceWindowResize = false)

```

Comment:
Omit docs and default argument on implementation

Reply:
Are you sure? The majority of calls are from within `MainForm` itself, not through the interfaces

Reply:
IIRC if you omit the default argument you're sometimes forced to do `((IIntf) this).FrameBufferResized()` nonsense. I'm of the opinion that it should be explicit at every callsite anyway, but for this PR having it duplicated isn't a problem, it'll get fixed later.

Definitely drop the doc comment though.